documentation/src/docs/reader-group-design.md

```diff
@@ -9,85 +9,90 @@ You may obtain a copy of the License at
 
-- a map of online readers to the segments they own
-- a list of positions in segments that can be taken over.
+ - A map of online Readers to the segments they own.
+ - A list of positions in segments that can be taken over.
 
-Every time the Readers in a ReaderGroup change, the state can be updated. Similarly each time one of the readers is going to start reading from a new segment, it can update the replicated state. 
-This allows all readers to know about all the other Readers in their ReaderGroup and which segments they own.
+Every time the Readers in a Reader Group change, the state can be updated. The replicated state is updated every time when one of the Readers starts reading from a new segment. Thus every Reader is aware of the segments owned by all the Readers in the their Reader Group.
 
 Given this information:
 
-- A new reader can infer which segments are available to read from. (By virtue of it being absent from the state)
-- Dealing with a segment being merged becomes easy, because the last reader to reach the end of its pre-merge segment knows it can freely take ownership of the new segment.
-- Readers can see their relative load and how they are progressing relative to the other readers in their group and can decide to transfer segments if things are out of balance.
-- This allows readers to take action directly to ensure all the events are read without the need for some external tracker.
+ - A new Reader can infer which segments are available to read from. (By virtue of it being absent from the state).
+ - Merging segments becomes easy, because the last Reader to reach the end of its pre-merge segment knows it can freely take ownership of the new segment.
+ - Readers can see their relative load and how they are progressing relative to the other Readers in their group and can decide to transfer segments if things are out of balance.
+ - This allows Readers to take action directly to ensure all the events are read without the need for some external tracker.
+
+## Reader Group APIs
+
+The external APIs to manage Reader Groups could be added to the `StreamManager` object. It consist of:
 
-## ReaderGroup APIs
-The external APIs to manage ReaderGroups could be added to the StreamManager object. They consist of:
 ```
     ReaderGroup createReaderGroup(String name, Stream stream, ReaderGroupConfig config);
     ReaderGroup getReaderGroup(String name, Stream stream);
     void deleteReaderGroup(ReaderGroup group);
 ```
-When a ReaderGroup is created, it creates a [StateSynchronizer](state-synchronizer-design.md) shared by the readers. To join a ReaderGroup readers would just specify it in their configuration:
+When a Reader Group is created, it creates a [State Synchronizer](state-synchronizer-design.md) shared by the Readers. To join a Reader Group, Readers would just specify it in their configuration:
```

Comment:
as I understand we should not have a space in between `State Synchronizer`. ref: https://github.com/pravega/pravega/blob/master/documentation/src/docs/state-synchronizer-design.md

nitpick : Also `Readers` should be changed to readers ( minus capital letter) right? 
We could follow the same pattern at other places too...

Reply:
@shrids  The naming conventions, we are still working on ...as it is not consistent throughout all the 32 docs....
Hence i have planned to update the terminoloy.md  post beta. Is that sounds good!